pkg/controller/mutatorstatus/mutatorstatus_controller.go

```diff
@@ -124,7 +124,12 @@ func PodStatusToMutatorMapper(selfOnly bool, kindMatch string, packerMap handler
 			}
 		}
 		u := &unstructured.Unstructured{}
-		u.SetGroupVersionKind(schema.GroupVersionKind{Group: v1beta1.MutationsGroup, Version: "v1", Kind: kind})
+		// AssignImage is the only mutator in v1alpha1 still
+		v := "v1"
+		if kind == "AssignImage" {
+			v = "v1alpha1"
+		}
```

Comment:
Logging this would be very spammy (this gets run every time we update a MutatorPodStatus). It's also not that useful to log IMO --it would just tell us if we are reconciling an AssignImageMutator or any other mutator. Due to the high QPS of this reconciler, I think we should only log critical errors.